elephant/asset.py

```diff
@@ -78,7 +78,7 @@ def _signals_same_tstart(signals):
     signals : list
         a list of signals (e.g. AnalogSignals or SpikeTrains) having
         attribute `t_start`
-
+en
```

Comment:
What is this?

Reply:
It's a typo. If you look at my second commit it should have been removed. Please squash when merging.

Reply:
Ah, upper commit is the later one, that I didn't notice.
OK, I have no more comments now.